docs/src/dev/contributing.md

```diff
@@ -0,0 +1,19 @@
+
+# Contributing 
+
+```@contents
+Pages=["contributing.md"]
+```
+## Improve the documentation
+If you have ever been reading the documentation and been unsure about a description or instruction, the documentation can be improved. Since you are the person finding this gap in the documentation, you are also the person in the best position to fix it.
+
+The documentation is written in Markdown and built using [Documenter.jl](https://documenter.juliadocs.org/stable/man/guide/). The source code for all the docs is [here](https://github.com/numlinalg/RLinearAlgebra.jl/tree/master/docs). 
+
+
+## Bug Reports
+If you find a bug in our software we would love to know about it. You can make an issue relating to a bug report [here](https://github.com/numlinalg/RLinearAlgebra.jl/issues/new?assignees=dmaldona%2C+npritch928%2C+vp314&labels=bug&projects=&template=bug_report.md&title=). 
+
+## Feature Requests
+Have a new method that you think would be a valuable addition to the package? Make a [feature request](https://github.com/numlinalg/RLinearAlgebra.jl/issues/new?assignees=dmaldona%2C+npritch928%2C+vp314&labels=enhancement&projects=&template=feature_request.md&title=) and let us know about it. Please provide details of why this feature would be valuable and if possible point us to resources to help us better understand the feature.
+
+
```

Comment:
@nathanielpritchard I think we want casual collaborators to fork the repo and create a new branch. Can you include a link to this workflow? We would like them to do like here
https://jump.dev/JuMP.jl/stable/developers/contributing/